### PR TITLE
Add support for parallel task execution

### PR DESCRIPTION
**Pull request by xjunior**
Closes https://github.com/powerhome/cobra_commander/issues/39

### Multiple components

ezgif-6-939acb64b9a8

### Single component

Aug-21-2020 19-33-31

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Add support for parallel task execution to `cobra exec`.
```
Comment (benlangfeld) [inline CHANGELOG.md:5]: Is the title of this PR wrong?

File: cobra_commander.gemspec [hunk 2]
```diff
@@ -37,6 +37,8 @@ DESCRIPTION
   spec.add_dependency "bundler", "~> 1.17"
   spec.add_dependency "thor", ["< 2.0", ">= 0.18.1"]
   spec.add_dependency "ruby-graphviz", "~> 1.2.3"
+  spec.add_dependency "tty-command", "~> 0.9.0"
```
Comment (benlangfeld) [inline cobra_commander.gemspec:40]: Is this part of parallel task execution?

Comment (xjunior) [reply cobra_commander.gemspec:40]: Yes

**Comment by xjunior** [comment]
@benlangfeld @terryfinn thoughts? Shall we go ahead and merge this?

**Comment by xjunior** [comment]
Closed in favor of https://github.com/powerhome/cobra_commander/pull/49

**Pull request closed without merging**